### ISSUE TITLE
Screen Reader-Projects-My Stuff-Import: Narrator focus is not Synchronised with keyboard focus on Import dialog box.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2524]
**Pre-Requisite:** Turn on Narrator(Win + Ctrl + Enter)

**User Experience:** 
Users who depends on Screen Reader will get confuse if Narrator focus is not Synchronised with keyboard focus.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Projects control lying in the header section on the page and select it.
4. Navigate to the MY Stuff tab on the page opened and select Import option.
5. Navigate to various controls lying on the page opened.
6. Verify that Narrator focus is synchronised with keyboard focus or not.
 
**Actual Result:-**
Narrator focus is not synchronised with keyboard focus on Import dialog box.

**Expected Result:-**
Narrator focus should be synchronised with keyboard focus on Import dialog box so that screen reader can provide correct information about the Elements of the web page.

**Note:** This issue does not exists with NVDA +FF.
 
**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b24201d1-2276-4471-8183-e1e5353031ad}

**Please refer the attachment.**
[MAS4.2.10_Projects_Import.file.zip](https://github.com/Microsoft/pxt/files/1179801/MAS4.2.10_Projects_Import.file.zip)

**Comment by veler** [comment]
After investigation : I've found how to give the focus back after selecting a file, but it looks like the other described issues are linked to some native behavior. Same reproduction here : [https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_fileupload_get](https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_fileupload_get)

Suggested fix @microsoftsam  : make a custom input field with a custom Browse button

**Comment by veler** [comment]
@v-amik Issue fixed and available to test

**Comment by v-amik** [comment]
As per process, bug should be assigned to HCL for regression

**Comment by veler** [comment]
For the Browse button, it's a native component of Windows and is out of our control.

**Comment by v-mesala** [comment]
Verified on URL:  https://makecode.microbit.org/acc and issue is not fixed. Hence reactivating the bug.

**Comment by v-mesala** [comment]
added active tag

**Comment by veler** [comment]
For the Browse button, it's a native component of Windows and is out of our control. Marked as external and need additional information to be sure that @v-mesala is talking about the Browse button.

**Comment by veler** [comment]
@abchatra , please forward this issue to the MSEdge team.

**Comment by abchatra** [comment]
Edge issue.

**Comment by v-swasha** [comment]
As per the product team comment bug is External to Edge .Hence logged a bug under Edge VSO for further tracking : Bug 18190514(https://microsoft.visualstudio.com/OS/_workitems/edit/18190514)
